examples/app-java/src/main/java/ly/iterative/example/AppJava.java

```diff
@@ -29,7 +29,7 @@ public static void main(String[] args) {
             Context.VALID_ONLY_REQUIRED_PROPS,
             new Options(
                 environment,
-                new ArrayList<>(Arrays.asList(
+                new ArrayList<Plugin>(Arrays.asList(
```

Comment:
@dantetam the fact that this works in Java gives me hope there is a better way to support generics in Kotlin. Something interesting I came across is a [comment at the bottom of this article](https://kotlinlang.org/docs/typecasts.html#unchecked-casts) that menstions Java retains the type but they are erased in Kotlin's Lists. I wonder if using a non-List collection here could help us.